openquakeplatform/bin/deploy.sh

```diff
@@ -0,0 +1,303 @@
+sudo -u postgres psql -d $GEO_DBUSER-imports -c 'GRANT ALL ON spatial_ref_sys TO PUBLIC;'
+
+# add unaccent extension and icompare_unaccent function into postgres
+cat << EOF | sudo -u postgres psql -d geonode                                                                                                                                                                                         
+    -- NOTE: originally deployed as migration, we realized that this sql script must
+    --       be executed for every new installation (devel or production).
+    --       The script is idempotent so we decided to keep the original migration script too
+    
+    DROP OPERATOR IF EXISTS =~@ (character varying, character varying);
+    DROP FUNCTION IF EXISTS icompare_unaccent(character varying, character varying);
+    DROP EXTENSION IF EXISTS unaccent;
+    
+    CREATE EXTENSION unaccent;
+    CREATE FUNCTION icompare_unaccent(character varying, character varying) RETURNS boolean
+        AS 'SELECT upper(unaccent(\$1)) LIKE upper(unaccent(\$2));'
+        LANGUAGE SQL
+        IMMUTABLE
+        RETURNS NULL ON NULL INPUT;
+   
+    CREATE OPERATOR =~@ (
+        LEFTARG = character varying,
+        RIGHTARG = character varying,
+        PROCEDURE = icompare_unaccent,
+        NEGATOR = !=~@
+    );
+EOF
+
+#insert line in pg_hba.conf postgres
+sudo sed -i '1 s@^@local  all             '"$GEO_DBUSER"'             md5\n@g' /etc/postgresql/9.5/main/pg_hba.conf
+sudo sed -i '2 s@^@host  all    '"$GEO_DBUSER"'         '"$LXC_IP"'/32             md5\n@g' /etc/postgresql/9.5/main/pg_hba.conf
+# sudo sed -i '2 s@^@host  all    '"$GEO_DBUSER"'         '"$LXC_IP"'             md5\n@g' /etc/postgresql/9.5/main/pg_hba.conf
+sudo sed -i "1 s@^@listen_addresses = '127.0.0.1,localhost,"$LXC_IP"'\n@g" /etc/postgresql/9.5/main/postgresql.conf
+
+#restart postgres
+sudo service postgresql restart
+}
+
+function clone_platform() {
+    # clone oq-platform
+    cd $HOME
+    git clone https://github.com/gem/oq-platform2.git
+    cd $GIT_REPO
+    git checkout oqstyle
+    pip install -e .
+}
+
+function oq_application() {
+    # clone ipt, taxtweb, building-classification-survey
+    cd $HOME
+    for repo in oq-platform-taxtweb oq-platform-ipt oq-platform-building-class oq-platform-data; do
+        # for repo in oq-platform-taxtweb; do
+        if [ "$GIT_BRANCH" = "master" ]; then false ; else git clone -b "$GIT_BRANCH" https://github.com/gem/${repo}.git ; fi || git clone -b $GIT_REPO https://github.com/gem/${repo}.git || git clone https://github.com/gem/${repo}.git
+        if [ "${repo}" != "oq-platform-data" ]; then
+            cd ${repo}
+            pip install -e .
+            cd $HOME
+        fi
+    done
+}
+
+function install_geonode() { 
+    # clone geonode
+    cd $HOME
+    git clone --depth=1 -b "$GIT_GEO_REPO" https://github.com/GeoNode/geonode.git
+
+    # download Geonode zip
+    wget http://ftp.openquake.org/$GIT_REPO/GeoNode-2.6.x.zip
```

Comment:
It's unclear to me why you first download GeoNode from GitHub via git and then you re-download geonode as a zip file.